src/genie/libs/parser/iosxe/show_snmp.py

```diff
@@ -79,3 +79,153 @@ def cli(self, output=None):
+            Optional("storage_type"): str,
+            "readview": str,
+            "writeview": str,
+            "notifyview": str,
+            "row_status": {
+                "status": str,
+                Optional("access_list"): str
+            }
+        }
+
+        
+    }
+
+
+# ====================
+# Parser for:
+#  * 'show snmp group'
+# ====================
+class ShowSnmpGroup(ShowSnmpGroupSchema):
+    """Parser for show snmp group"""
+
+    cli_command = 'show snmp group'
+
+    def cli(self, output=None):
+        if output is None:
+            output = self.device.execute(self.cli_command)
+        else:
+            output = output
+
+        # groupname: 2c                               security model:v1 
+        # contextname: <no context specified>         storage-type: volatile
+        # readview : <no readview specified>          writeview: <no writeview specified>        
+        # notifyview: *tv.FFFF58bf.eaFF58bf.eaFFFFFF.F
+        # row status: active
+        #
+        # groupname: ag-ro                           security model:v1 
+        # contextname: <no context specified>         storage-type: volatile
+        # readview : v1default                        writeview: <no writeview specified>        
+        # notifyview: *tv.FFFF58bf.eaFF58bf.eaFFFFFF.F
+        # row status: active
+        #
+        # groupname: AlfaV                    security model:v2c 
+        # contextname: <no context specified>         storage-type: permanent
+        # readview : v1default                        writeview: <no writeview specified>        
+        # notifyview: <no notifyview specified>       
+        # row status: active	access-list: 90
+
+        # groupname: 2c                               security model:v1
+        p_group = re.compile(r"^groupname:\s+(?P<group>\S+)\s+security model:(?P<security>.*)$")
+
+        # contextname: <no context specified>         storage-type: permanent
+        p_context_none = re.compile(r"^contextname:\s+<no context specified>\s+storage-type:\s+(?P<storage>\S+)$")
+        p_context = re.compile(r"^contextname:\s+(?P<context>\S+)\s+storage-type:\s+(?P<storage>\S+)$")
+
+        # readview : <no readview specified>          writeview: <no writeview specified>   
+        p_rw_none = re.compile(r"^readview :\s+<no readview specified>\s+writeview:\s+<no writeview specified>$")
+        p_r_wnone = re.compile(r"^readview :\s+(?P<read>\S+)\s+writeview:\s+<no writeview specified>$")
+        p_rw = re.compile(r"^readview :\s+(?P<read>\S+)\s+writeview:\s+(?P<write>\S+)$")
+
+        # notifyview: <no notifyview specified>
+        p_notify_none = re.compile(r"^notifyview: <no notifyview specified>$")
+        p_notify = re.compile(r"^notifyview:\s+(?P<notify>\S+)$")
+
+        # row status: active	access-list: 90
+        p_row = re.compile(r"^row\s+status:\s+active$")
+        p_row_ac = re.compile(r"^row\s+status:\s+active\s+access-list:\s+(?P<access>\S+)$")
```

Comment:
for each regex pattern, can you put the corresponding output above. If `p_row_ac ` captures dat aslightly different, please place it above. Same for above patterns.

Reply:
Added.